src/components/Popover/index.tsx

```diff
@@ -0,0 +1,99 @@
+'use client'
+
+import * as React from 'react'
+import * as PopoverPrimitive from '@radix-ui/react-popover'
+import * as Separator from '@radix-ui/react-separator'
+import { cva, type VariantProps } from 'class-variance-authority'
+import * as SeparatorPrimitive from '@radix-ui/react-separator'
+import * as ScrollArea from '@radix-ui/react-scroll-area'
+
+import { cn } from '@/lib/utils'
+
+const popoverVariants = cva([
+  'z-50',
+  'border border-[1px] border-border-subtle dark:border-border-dark',
+  'rounded-lg',
+  'bg-background dark:bg-background-dark',
+  'text-foreground text-sm font-normal dark:text-white',
+  'shadow',
+  'outline-none',
+  'overflow-scroll',
+  'data-[state=open]:animate-in',
+  'data-[state=closed]:animate-out',
+  'data-[state=closed]:fade-out-0',
+  'data-[state=open]:fade-in-0',
+  'data-[state=closed]:zoom-out-95',
+  'data-[state=open]:zoom-in-95',
+  'data-[side=bottom]:slide-in-from-top-2',
+  'data-[side=left]:slide-in-from-right-2',
+  'data-[side=right]:slide-in-from-left-2',
+  'data-[side=top]:slide-in-from-bottom-2'
```

Comment:
Do these animations work?

Reply:
No they don't, there is no mention of animations in the figma though, so I have removed